cranelift/codegen/src/isa/x86/unwind.rs

```diff
@@ -215,11 +336,37 @@ impl UnwindInfo {
             return None;
         }
 
+        if saved_fpr {
+            if static_frame_allocation_size > 240 && saved_fpr {
+                panic!("stack frame is too large ({} bytes) to use with Windows x64 SEH when preserving FPRs", static_frame_allocation_size);
```

Comment:
Given users might run into this panic on Windows until we fix the underlying issue (granted, without the panic they weren't getting proper register restores or unwind which is a more severe problem), perhaps we should note that this is a known limitation and link to the GitHub issue describing the problem?

Reply:
This is a good idea. I'll add that in once I have an issue filed.